SharpPcap/ICaptureHeader.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SharpPcap
+{
+    public interface ICaptureHeader
+    {
+        PosixTimeval Timeval { get; }
```

Comment:
No docs?

Reply:
let me add something, it wasn't clear the value of the comments and nothing came to mind so I left that info out